### PR TITLE
 add removeNextflowVersion function

### PR DESCRIPTION
**Pull request by maxulysse**
Sorry, made a mess with my branches, so recreating the PR here, it replaces #3

### CODE REVIEW

**Comment by sateeshperi** [comment]
is the `tests/utils/nextflow.config` empty file needed ?

File: src/main/java/nf_core/nf/test/utils/Methods.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import org.yaml.snakeyaml.Yaml;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Map;
+
+public class Methods {
+  public static Map<String, Map<String, Object>> readYamlFile(String filePath) {
+    Yaml yaml = new Yaml();
+
+    System.err.println(filePath);
```
Comment (nvnieuwk) [inline src/main/java/nf_core/nf/test/utils/Methods.java:12]:
```suggestion
```

**Review by nvnieuwk — APPROVED** [review]
LGTM! Maybe this can later be expanded to be able to customize which fields should be removed, but let's do that another time :p

File: src/main/java/nf_core/nf/test/utils/Methods.java (hunk 1 above)
Comment (maxulysse) [reply src/main/java/nf_core/nf/test/utils/Methods.java:12]: yeah I was thinking the same, but that's a good first start already

**Comment by maxulysse** [comment]
> LGTM! Maybe this can later be expanded to be able to customize which fields should be removed, but let's do that another time :p

yeah I was thinking the same, but that's a good first start already

**Comment by maxulysse** [comment]
> is the `tests/utils/nextflow.config` empty file needed ?

yeah, otherwise nf-test/nextflow complains about it

**Pull request merged**